products/deploymentmanager/terraform.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Deployment: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "projects/{{project}}/deployments/{{name}}"
+    import_format: ["projects/{{project}}/deployments/{{name}}"]
+    timeouts: !ruby/object:Api::Timeouts
+      # Very long to support very long deployments
+      insert_minutes: 60
+      update_minutes: 60
+      delete_minutes: 60
+    docs: !ruby/object:Provider::Terraform::Docs
+      warning: |
+        Deployment Manager shares similar behavior with Terraform as both
+        products manage GCP resource lifecycle and state. This Terraform
+        resource is intended only to manage a Deployment resource,
+        and attempts to manage the Deployment's resources in Terraform as well
+        will likely result in errors or unexpected behavior as the two tools
+        fight over ownership. We strongly discourage doing so unless you are an
+        experienced user of both tools.
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "deployment_manager_deployment_basic"
+        primary_resource_id: "deployment"
+        primary_resource_name: "fmt.Sprintf(\"my-deployment-%s\", context[\"random_suffix\"])"
+        vars:
+          deployment_name: "my-deployment"
+          service_account_id: "tf-dm-account"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "deployment_manager_deployment_imports"
+        primary_resource_id: "deployment"
+        skip_test: true
+    properties:
+      createPolicy: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
```

Comment:
Is this needed? `url_param_only` should make it redundant. If it's for the example ISV, we can include `url_param_only` fields there too.

Reply:
i think this was left over from when I was figuring out that ISV didn't include url_param_only. I can delete.

This does actually bring up a seperate point (and maybe a new PR) - i think we should lift ignore_read into the main type.rb and get rid of url_param_only, which is a combination of ignore_read and "don't send in the request body". Isn't that what parameters are? I'd love to have a ignore_read method that returns `@ignore_read || @url_param_only` or something, but we can't do that on Terraform property_override

Reply:
Does settings something in `parameters` instead of `properties` do anything? I think `url_param_only` is the one actually doing something. 

Cleaning all that up in a separate PR would be a good idea, and ignore_read makes a lot of sense in `api.yaml`.

Reply:
🤷‍♀lol we should clean something up if it's like haha what does parameters do??